webknossos/webknossos/dataset/layer.py

```diff
@@ -1034,9 +1034,9 @@ def _setup_mag(self, mag: Mag) -> None:
                 info.compression_mode,
             )
             self._mags[mag]._read_only = self._dataset.read_only
-        except ArrayException as e:
-            logging.error(
-                f"Failed to setup magnification {mag_name}, which is specified in the datasource-properties.json. See {e}"
+        except ArrayException:
+            logging.exception(
+                f"Failed to setup magnification {mag_name}, which is specified in the datasource-properties.json:"
```

Comment:
The colon makes me think something comes afterwards, but the method ends here, is there some magic going on? :D

Reply:
Yes :D Afterwards, the current exception is logged as well, some black magic :magic_wand: 
https://docs.python.org/3/library/logging.html#logging.exception

Reply:
Ah, I missed the `.exception`. That’s not all that implicit then, fair enough 👍